keyboards/sofle/keymaps/default/keymap.c

```diff
@@ -0,0 +1,403 @@
+#include QMK_KEYBOARD_H
+
+enum sofle_layers {
+    /* _M_XYZ = Mac Os, _W_XYZ = Win/Linux */
+    _QWERTY,
+    _COLEMAK,
+    _LOWER,
+    _RAISE,
+    _ADJUST,
+};
+
+enum custom_keycodes {
+    KC_QWERTY = SAFE_RANGE,
+    KC_COLEMAK,
+    KC_LOWER,
+    KC_RAISE,
+    KC_ADJUST,
+    KC_PRVWD,
+    KC_NXTWD,
+    KC_LSTRT,
+    KC_LEND,
+    KC_DLINE
+};
+
+#define ______ KC_TRNS
+#define XXXXXX KC_NO
+
```

Comment:
We should use the ones provided by QMK here.

```suggestion

```

Reply:
Thanks for the suggestion, done.